test/language/statements/await-using/global-use-before-initialization-in-declaration-statement.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2023 Ron Buckton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-declarative-environment-records-getbindingvalue-n-s
+description: >
+    await using: global use before initialization in declaration statement.
+    (TDZ, Temporal Dead Zone)
+negative:
+  phase: runtime
+  type: ReferenceError
+flags: [async]
+includes: [asyncHelpers.js]
+features: [explicit-resource-management]
+---*/
+
+asyncTest(async function () {
+  await using x = x + 1;
```

Comment:
Given that this is a runtime error, it results in a rejection on the async function's promise.

According to the `asyncTest` definition, a rejection directly results in a failure.
So I suppose you need to handle the rejection?

https://github.com/tc39/test262/blob/fd594a077a0a018440f241fdd421a5862f1153f5/harness/asyncHelpers.js#L17-L31
https://github.com/tc39/test262/blob/fd594a077a0a018440f241fdd421a5862f1153f5/harness/doneprintHandle.js#L12-L17